sdks/python/apache_beam/ml/transforms/tft.py

```diff
@@ -291,6 +291,41 @@ def apply_transform(
     return output_dict
 
 
+@register_input_dtype(float)
+class ScaleToGaussian(TFTOperation):
+  def __init__(
+      self,
+      columns: List[str],
+      elementwise: bool = False,
+      name: Optional[str] = None):
+    """
+    This function applies a scaling transformation on the given columns
+    of incoming data. The operation transforms the input column values
+    to an approximately normal distribution with mean 0 and variance of 1.
+    The Gaussian transformation is only applied if the column has long tails;
```

Comment:
Does this mean: "do not use this operation if your data doesn't have long tails, use z-score instead" or does it mean: "this transform is doing some sophisticated normalization but if your data doesn't have long tails, then it's the same as z-score normalization"?

Reply:
Consider adding more details from the below link and/or consider adding: "For more information, see: https://www.tensorflow.org/tfx/transform/api_docs/python/tft/scale_to_gaussian"

Reply:
made a few changes to clarify, it's the latter